funnel/templates/js/server_error.js.jinja2

```diff
@@ -0,0 +1,6 @@
+{% macro error_txt() %}
+  {% raw %}
+    <span>{{ gettext('An internal server error occurred. Our support team has been notified and will investigate.') }}</span>
+    <span>{{ gettext('Unable to connect. Please reload and try again.') }}</span>
```

Comment:
I think this one should use a link to reload the page. Avoid use of the words "please" and "kindly" -- these are Indianisms. This text reads better as "Unable to connect. Check connection and tap to reload"